gulpfile.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var path = require('path');
+var gulp = require('gulp');
+var mocha = require('gulp-mocha');
+var istanbul = require('gulp-istanbul');
+var nsp = require('gulp-nsp');
+var plumber = require('gulp-plumber');
+//var exec = require('child_process').exec;
+var jshint = require('gulp-jshint');
+
+gulp.task('nsp', function (cb) {
+    nsp({package: path.resolve('package.json')}, cb);
+});
+
+gulp.task('pre-test', function () {
+    return gulp.src('lib/**/*.js')
+        .pipe(istanbul({
+            includeUntested: true
+        }))
+        .pipe(istanbul.hookRequire());
+});
+
+/*gulp.task('startHazelcast', function (cb) {
```

Comment:
not fully implemented because it's not very clear how to bring jars, shutdown client after, etc. Just have it there as prototype.